### ISSUE TITLE
train / test module

### ISSUE THREAD

**Issue by andreistirb** [andreistirb/autonomous_driving_kaggle_challenge#4]
Implement everything related to training the network - loss functions, optimizers, etc.

**Comment by tiberiu92** [comment]
This would be the structure of the neural network ? (input, hidden layers, output)

**Comment by andreistirb** [comment]
@tiberiu92 wrong issue maybe? I guess you mean #3

**Comment by tiberiu92** [comment]
Yes. My mistake.
